packages/storybook/stories/MessageThread/MessageThread.stories.tsx

```diff
@@ -350,10 +350,32 @@ const MessageThreadStory = (args): JSX.Element => {
   };
   const [galleryImages, setGalleryImages] = useState<Array<ImageGalleryImageProps> | undefined>(undefined);
 
-  const onInlineImageClicked = (attachment: FileMetadata): Promise<void> => {
+  const onInlineImageClicked = (attachmentId: string, messageId: string): Promise<void> => {
+    const messages = chatMessages?.filter((message) => {
+      return message.messageId === messageId;
+    });
+    if (!messages || messages.length <= 0) {
+      return Promise.reject(`Message not found with messageId ${messageId}`);
+    }
+    const chatMessage = messages[0] as ChatMessage;
+
+    const attachments = chatMessage.attachedFilesMetadata?.filter((attachment) => {
+      return attachment.id === attachmentId;
+    });
+
+    if (!attachments || attachments.length <= 0) {
+      return Promise.reject(`Attachment not found with id ${attachmentId}`);
+    }
+
+    const attachment = attachments[0];
+    attachment.name = chatMessage.senderDisplayName || '';
     const title = 'Message Thread Image';
+    const titleIcon = (
+      <Persona text={chatMessage.senderDisplayName} size={PersonaSize.size32} hidePersonaDetails={true} />
```

Comment:
nitpick: break into multiple lines for readability

Reply:
> nitpick: break into multiple lines for readability

Prettifier automatically set it back to one line. It might be at the breaking point according to our linter rule.